scipy/special/add_newdocs.py

```diff
@@ -737,22 +737,72 @@ def add_newdoc(name, doc):
     """)
 
 add_newdoc("betainc",
-    """
-    betainc(a, b, x)
+    r"""
+    betainc(a, b, x, out=None)
 
-    Incomplete beta integral.
+    Incomplete beta function.
 
-    Compute the incomplete beta integral of the arguments, evaluated
-    from zero to `x`::
+    Computes the incomplete beta function, defined as [1]_:
+    
+    .. math::
 
-        gamma(a+b) / (gamma(a)*gamma(b)) * integral(t**(a-1) (1-t)**(b-1), t=0..x).
+        I_x(a, b) = \frac{\Gamma(a+b)}{\Gamma(a)\Gamma(b)} \int_0^x 
+        t^{a-1}(1-t)^{b-1}dt,
+        
+    for :math:`0 \leq x \leq 1`.
 
+    Parameters
+    ----------
+    a, b : array-like
+           Positive, real-valued parameters
+    x : array-like
+        Real-valued such that :math:`0 \leq x \leq 1`,
+        the upper limit of integration
+    out : ndarray, optional
+        Optional output array for the function values
+    
+    Returns
+    -------
+    array-like
+        Value of the incomplete beta function
+    
     Notes
     -----
-    The incomplete beta is also sometimes defined without the terms
-    in gamma, in which case the above definition is the so-called regularized
-    incomplete beta. Under this definition, you can get the incomplete beta by
-    multiplying the result of the scipy function by beta(a, b).
+    The incomplete beta function is also sometimes defined 
+    without the terms in gamma, in which case the above
```

Comment:
Perhaps "... without the Gamma terms ..."?

Reply:
Sure. I also made it link to the gamma doc using backticks.